src/components/__tests__/input.test.tsx

```diff
@@ -0,0 +1,74 @@
+import { render } from '@testing-library/react'
+import React from 'react'
+
+import { Input } from '@/components/input'
+
+describe('Input', () => {
+  it('should render an input with the `aria-label` attribute', () => {
+    const { queryByLabelText } = render(<Input aria-label="test input" />)
+
+    expect(queryByLabelText('test input')).toBeInTheDocument()
+    expect(queryByLabelText('test input')).toHaveAttribute('aria-label', 'test input')
+  })
```

Comment:
I don't think we need to the `aria-label` attribute for the input, since the label will be available in the DOM.